data/json/items/magazine/22.json

```diff
@@ -34,6 +34,78 @@
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "22": 165 } } ]
   },
+  {
+    "id": "a180mag1",
```

Comment:
This magazine is identical to the "a180mag", it should probably at least weigh less as it is made from polymer rather than steel.

Reply:
Good point; however, the issue with that is in changing the weight of the magazines caused the ranged balance test to have a stroke. I'll attempt to change it anyway, but I don't know if it'll work.